ingest/build-configs/ncbi/defaults/host-map.tsv

```diff
@@ -50,6 +50,8 @@ harris hawk	Avian
 harris-hawk	Avian
 hawk	Avian
 Homo sapiens	Human
+HOSP	Environment
```

Comment:
I'm assuming `HOSP` stands for "hospital" here. Please chime in if this stands for some obscure host name that I'm not aware of.

Reply:
HOSP = house sparrow

Funnily enough there's a semi-official set of "bird codes". Here's the list: https://www.carolinabirdclub.org/bandcodes.html. May be useful to link from the ingest readme (or from `host-map.tsv`.

Reply:
Ah, thank you! Will link to it from the `host-map.tsv`.

Reply:
Updated in https://github.com/nextstrain/avian-flu/pull/64/commits/1297ecd1d15808ff6e1a7b6a43b4bd0ef943bf62